src/eval.c

```diff
@@ -1508,9 +1508,9 @@ void ldbEval(lua_State *lua, sds *argv, int argc) {
     lua_pop(lua,1);
 }
 
-/* Implement the debugger "redis" command. We use a trick in order to make
+/* Implement the debugger "server command". We use a trick in order to make
```

Comment:
I've never used the Lua debugger in Redis so I don't really know what this is supposed to mean, but reading the source code below, I believe it calls the Lua function `redis.call`. We are adding a synonym for that called `server.call` but the debugger implementation below still uses redis.call and honestly i don't think it matters. Anyway, this PR is only about the comments so... perhaps it's just the quotation mark that's a bit off:

```suggestion
/* Implement the debugger "server" command. We use a trick in order to make
```